drivechain-server/api/wallet/wallet.go

```diff
@@ -0,0 +1,212 @@
+package api_wallet
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"reflect"
+	"sync/atomic"
+	"time"
+
+	"connectrpc.com/connect"
+	"github.com/LayerTwo-Labs/sidesail/drivechain-server/bdk"
+	pb "github.com/LayerTwo-Labs/sidesail/drivechain-server/gen/wallet/v1"
+	rpc "github.com/LayerTwo-Labs/sidesail/drivechain-server/gen/wallet/v1/walletv1connect"
+	corepb "github.com/barebitcoin/btc-buf/gen/bitcoin/bitcoind/v1alpha"
+	coreproxy "github.com/barebitcoin/btc-buf/server"
+	"github.com/btcsuite/btcd/btcutil"
+	"github.com/rs/zerolog"
+	"github.com/samber/lo"
+	"google.golang.org/protobuf/types/known/emptypb"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+var _ rpc.WalletServiceHandler = new(Server)
+
+// New creates a new Server with external services registered
```

Comment:
This comment is wrong. No registering here!

Reply:
hmmm

Reply:
true!